### ISSUE TITLE
Persistent Connection

### ISSUE THREAD

**Issue by lifeofguenter** [gitterHQ/irc-bridge#22]
How can I get a persistent username/password combination to allow a persistent connection?

Currently I need to generate a password each time by visiting irc.gitter.im - because the old password won't work if my IP changes.

**Comment by malditogeek** [comment]
What problem are exactly you having?

The only thing you need to connect to the bridge is the token (sent as server password) and that should work regardless of your IP.

Mauro.

**Comment by lifeofguenter** [comment]
When I connect from work it works :) - when I connect (with the same 
credentials) from home - it does not work. Sometimes I get the following 
message:

gitter>  Authentication failed. Get a valid token from https://irc.gitter.im

But in any case I am unable to join any channels (no error message).

**Comment by malditogeek** [comment]
What client are you using? Are you using the same client at work and home?

Just make sure you're sending the token as the server password on both
cases.

If you can add a screenshot of your config would be great (without your
credentials :))

On Mon, May 11, 2015 at 10:49 AM, Günter Grodotzki <notifications@github.com
> wrote:

> When I connect from work it works :) - when I connect (with the same
> credentials) from home - it does not work. Sometimes I get the following
> message:
>
> gitter> Authentication failed. Get a valid token from
> https://irc.gitter.im
>
> But in any case I am unable to join any channels (no error message).
>
>
> On 11/05/2015 11:15, Mauro Pompilio wrote:
> > What problem are exactly you having?
> >
> > The only thing you need to connect to the bridge is the token (sent as
> > server password) and that should work regardless of your IP.
> >
> > Mauro.
> >
> > > On 11 May 2015, at 06:55, Günter Grodotzki

**Comment by lifeofguenter** [comment]
Yes, I am using the same client - actually even the same laptop :)
